overrides/scripts/AbyssalCraft.zs

```diff
@@ -77,6 +77,9 @@ recipes.remove(<abyssalcraft:odb>);
 recipes.remove(<abyssalcraft:coraliumstone>);
 recipes.addShaped(<abyssalcraft:coraliumstone> * 4, [[<quantumflux:darkstone>,<abyssalcraft:ccluster9>,<quantumflux:darkstone>],[<abyssalcraft:ccluster9>,<ore:ingot_dark_soularium>,<abyssalcraft:ccluster9>],[<quantumflux:darkstone>,<abyssalcraft:ccluster9>,<quantumflux:darkstone>]]);
 
+# Abyssal Stone with Mystical Agriculture Essences
+recipes.addShaped(<abyssalcraft:stone:1>*16, [[<mysticalagriculture:coralium_essence>,<mysticalagriculture:stone_essence>,<mysticalagriculture:coralium_essence>],[<mysticalagriculture:stone_essence>,<mysticalagriculture:dirt_essence>,<mysticalagriculture:stone_essence>],[<mysticalagriculture:coralium_essence>,<mysticalagriculture:stone_essence>,<mysticalagriculture:coralium_essence>]]);
```

Comment:
uh, dirt essence? is that intentional?

Reply:
wasnt sure what to add in the middle, changed it to coralium